httpobs/database/database.py

```diff
@@ -180,10 +184,10 @@ def periodic_maintenance() -> int:
         return cur.rowcount
 
 
-def select_scan_grade_totals() -> dict:
-    # Used for /api/v1/getGradeDistribution
+def select_star_from(table: str) -> dict:
+    # Select all the rows in a given table. Note that this is specifically not parameterized.
```

Comment:
Why not? (No objections, but maybe the next line explaining went missing / doesn't exist?)